app/scripts/components/analysis/define/index.tsx

```diff
@@ -1,6 +1,13 @@
-import React from 'react';
+import React, {
```

Comment:
I'd say we can split each block (aoi, date, datasets) into a different file to have this one not be do long

Reply:
https://github.com/NASA-IMPACT/delta-ui/pull/284/commits/1eb5f54785a09fe6061c1463a6e98f83e013d33a